<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lamp Controller</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layoutPower">
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>LAMP X [Online]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTurnOn">
        <property name="toolTip">
         <string>Turn lamp on</string>
        </property>
        <property name="styleSheet">
         <string>background-color: lime;</string>
        </property>
        <property name="text">
         <string>On</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTurnOff">
        <property name="toolTip">
         <string>Turn lamp off</string>
        </property>
        <property name="styleSheet">
         <string>background-color: red; color: white;</string>
        </property>
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutBrightness">
      <item>
       <widget class="QLabel" name="labelBrightness">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Brightness [100%]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderBrightness">
        <property name="toolTip">
         <string>Adjust brightness (10-100%)</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutScene">
      <item>
       <widget class="QLabel" name="labelScene">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Light mode [None]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxScenes">
        <property name="toolTip">
         <string>Select a scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetScene">
        <property name="toolTip">
         <string>Apply selected scene</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutRGB">
      <item>
       <widget class="QLabel" name="labelRGB">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Light mode RGB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditR">
        <property name="toolTip">
         <string>Red (0–255)</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditG">
        <property name="toolTip">
         <string>Green (0–255)</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditB">
        <property name="toolTip">
         <string>Blue (0–255)</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetRGB">
        <property name="toolTip">
         <string>Apply RGB values</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
